internal/gengapic/testdata/foo_rest_client_init.want

```diff
@@ -0,0 +1,68 @@
+// internalFooClient is an interface that defines the methods availaible from Awesome Foo API.
+type internalFooClient interface {
+	Close() error
+	setGoogleClientInfo(...string)
+	Connection() *grpc.ClientConn
+	Zip(context.Context, *mypackagepb.Bar, ...gax.CallOption) (*mypackagepb.Foo, error)
+	ListLocations(context.Context, *google.cloud.locationpb.ListLocationsRequest, ...gax.CallOption) (*google.cloud.locationpb.ListLocationsResponse, error)
+	GetLocation(context.Context, *google.cloud.locationpb.GetLocationRequest, ...gax.CallOption) (*google.cloud.locationpb.Location, error)
+	SetIamPolicy(context.Context, *google.iam.v1pb.SetIamPolicyRequest, ...gax.CallOption) (*google.iam.v1pb.Policy, error)
+	GetIamPolicy(context.Context, *google.iam.v1pb.GetIamPolicyRequest, ...gax.CallOption) (*google.iam.v1pb.Policy, error)
+	TestIamPermissions(context.Context, *google.iam.v1pb.TestIamPermissionsRequest, ...gax.CallOption) (*google.iam.v1pb.TestIamPermissionsResponse, error)
+}
+
+// FooClient is a client for interacting with Awesome Foo API.
+// Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.
+type FooClient struct {
+	// The internal transport-dependent client.
+	internalFooClient
+
+	// The call options for this service.
+	CallOptions *FooCallOptions
+
+}
+
+// fooRestClient is a client for interacting with Awesome Foo API over REST transport.
+// It satisfies the internalFooClient interface.
+//
+// Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.
+type fooRestClient struct {
+	host string
+}
+
+func (c *fooRestClient) Zip(ctx context.Context, req *mypackagepb.Bar, opts ...gax.CallOption) (*mypackagepb.Foo, error) {
+	m := jsonpb.Marshaler{}
+	if jsonReq, err := m.MarshalToString(req); err != nil {
+		return nil, err
+	}
+}
+func (c *fooRestClient) ListLocations(ctx context.Context, req *google.cloud.locationpb.ListLocationsRequest, opts ...gax.CallOption) (*google.cloud.locationpb.ListLocationsResponse, error) {
```

Comment:
I didn't realize in the last PR but these request type package names are incorrect - I think a change in the test protos resulted in this. it should be just `locationpb` not `google.cloud.locationpb`. Same applies to the interface generation and other Unary methods. Can you fix this in this PR?

Reply:
We should've caught this, sorry: https://github.com/googleapis/gapic-generator-go/pull/577/files#diff-fb19ae54202b5f38b66ca11ddb0a56208a311c7deaa7acaaf10ec91f78595459R7-R11

Reply:
Of course.

Reply:
This is proving tricky to debug. Will you have some time tomorrow to do pair debugging?

Reply:
I think it's because of this: https://github.com/googleapis/gapic-generator-go/pull/577/files#diff-a1f28e41db88498f20bd1666eabe9c1336c3ca1c39fbac5e23b4cc80d1deb3b3R327

The GoPackage was set to the proto package, when it should be https://github.com/googleapis/googleapis/blob/master/google/iam/v1/iam_policy.proto#L29

Reply:
Sorry, I should've said something sooner, I had the tabs open looking at the diff

Reply:
We can of course pair on this though :)

Reply:
I'm going to tackle this separately. Carry on!